doc/api/Buffer_Control/setMaxBufferSize.md

```diff
@@ -0,0 +1,34 @@
+# setMaxBufferSize
+
+## Description
+
+Set the maximum memory the buffer can take up in the memory, in kilobytes
+Defaults at `Infinity`
+
+Once this value is reached, the player won't try to download new
+segments anymore.
+
+This feature was designed with devices that have limited memory and trying to play very
+high bitrates tracks in minds. 
+
+However on some custom targets, or just to better control the memory footprint
+of the player, you might want to set this limit.
+
+You can set it to `Infinity` to remove this limit and just let the browser do
+this job instead.
+
+<div class="warning">
+This option will have no effects if we didn't buffer at least <b>MIN_BUFFER_LENGTH</b> 
+<i>( defaults at 5sec )</i>
+</div>
```

Comment:
You can also add a warning to say that it hasn't any effect in directfile mode (see other documentation pages for the template).